src/main/webapp/Events.html

```diff
@@ -27,7 +27,7 @@
     </div>
     <div class="w3-container w3-light-grey w3-white w3-center" style="padding:128px 16px">
          <h1>BAResources events! </h1>
-    <div calendar><iframe id="calendar" src="https://calendar.google.com/calendar/embed?src=mlopezgarcia%40sps-program.com&ctz=America%2FMexico_City" width="800" height="600" frameborder="0" scrolling="no" ></iframe></div>
+    <div calendar><iframe src="https://calendar.google.com/calendar/embed?height=600&amp;wkst=1&amp;bgcolor=%23ffffff&amp;ctz=America%2FMexico_City&amp;src=Y19tNWd1Z2xnYTBkOGRsYzYza2QyZzg3YW5lZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t&amp;color=%23D50000&amp;title=Featured%20Events&amp;showCalendars=1&amp;showTabs=1&amp;showPrint=0&amp;showDate=1&amp;showNav=1" style="border:solid 1px #777" width="800" height="600" frameborder="0" scrolling="no"></iframe></div>
     </div>
```

Comment:
LGTM! Copy the footer from the other pages and add it here. It will make space between the calendar and end of the page.